src/skins/vector/css/common.css

```diff
@@ -161,8 +167,8 @@ input[type=text]:focus, textarea:focus {
     left: 0;
     width: 100%;
     height: 100%;
-    background-color: #000;
-    opacity: 0.2;
+    background-color: #e9e9e9;
```

Comment:
Apologies for that, I thought the `.mx_Dialog_background` was just for the Dialogs. I fixed it now, and just pushed